cinema/views.py

```diff
@@ -98,6 +94,30 @@
 
         return queryset.distinct()
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "title",
+                type=str,
+                description="Filter by title (ex. ?type=Titanic)",
+            ),
+            OpenApiParameter(
+                "genre",
+                type={"type": "array", "items": {"type": "number"}},
+                description="Filter by genres id (ex. ?facilities=2,3)",
+            ),
+            OpenApiParameter(
+                "actor",
+                type={"type": "array", "items": {"type": "number"}},
+                description="Filter by actors id (ex. ?facilities=2,3)",
```

Comment:
The parameter name in the `extend_schema` decorator should be `actors` instead of `actor` to match the query parameter used in the `get_queryset` method.